ext/ffi/ffi.stub.php

```diff
@@ -92,8 +92,8 @@ public function getStructFieldType(string $field_name): CType {}
 
     public function getFuncABI(): int {}
     public function getFuncReturnType(): CType {}
-    public function getFuncArgCount(): int {}
-    public function getFuncArgType(int $arg_index): CType {}
+    public function getFuncParameterCount(): int {}
+    public function getFuncParameterType(int $arg_index): CType {}
```

Comment:
arg->param in the param name as well?

Reply:
Good point, but there's only one index this can refer to, so $index without a prefix makes more sense and is often used elsewhere - https://www.php.net/manual/en/class.ziparchive.php just uses $index, not $zip_file_index , same for https://www.php.net/manual/en/splfixedarray.offsetget